app/models/tag.rb

```diff
@@ -12,8 +12,6 @@
 class Tag < ApplicationRecord
   has_and_belongs_to_many :statuses
 
-  HASHTAG_RE = /(?:^|[^\/\)\w])#([[:word:]_]*[[:alpha:]_][[:word:]_]*)/i
```

Comment:
I'm sorry. I will try to change the tag link generation process to use the regular expression of the mastodon.